packages/listbox/src/ListboxMixin.js

```diff
@@ -338,23 +338,29 @@ const ListboxMixinImplementation = superclass =>
     }
 
     /**
-     * When `multipleChoice` is false, will toggle, else will check provided index
-     * @param {number} index
-     * @param {'set'|'unset'|'toggle'} multiMode
+     * If an array is passed for multiple-choice, it will check the indexes in array, and uncheck the rest
+     * If a number is passed, the item with the passed index is checked without unchecking others
+     * For single choice, __onChildCheckedChanged we ensure that we uncheck siblings
+     * @param {number|number[]} index
      */
-    setCheckedIndex(index, multiMode = 'toggle') {
-      if (this.formElements[index]) {
-        if (!this.multipleChoice) {
+    setCheckedIndex(index) {
```

Comment:
I didn't understand the use cases and there were no tests, so I ended up removing it. For single choice you can do everything by 'set' because untargeted children get unchecked automatically. Toggle? just set to -1 to toggle it off, set to index to toggle it on. Unset? Just set to -1. 

I'm okay with adding it back in for multiple choice, as it would be useful there but maybe we should have tests so the behaviour is clearly specced, I couldn't really infer it from the implementation.

Reply:
Indeed, the 2nd param only makes sense for multipleChoice. We used it before in _listboxOnClick, so it was indirectly tested before. A unit test on top would be nice indeed.
We can also add it back once we have a use case for it, so we can get this going 👍